# Prediction Intervals

Recall from class that if $X_1,\ldots,X_{n+1}\sim\operatorname{Exp}(\lambda)$, then an exact two-sided $100(1-\alpha)\%$ prediction interval for $X_{n+1}$ is $\left(\dfrac{\overline x}{F_{\alpha/2, 2n, 2}}, \dfrac{\overline x}{F_{1-\alpha/2, 2n, 2}}\right)$.

Let's check this with $n=10$, $\lambda = 5$, and $1-\alpha=0.92$.

In [6]:
x <- rexp(10, 5)
mean(x)/qf(0.04, 20, 2, lower.tail=FALSE)
mean(x)/qf(0.96, 20, 2, lower.tail=FALSE)

[1] 0.006677098

[1] 0.6198424

So now we would expect that 92% of the time, an 11th element from this distribution would lie in the interval $(0.006677, 0.6198)$.  Let's check this with a Monte Carlo simulation.

In [8]:
mean(replicate(10000, {x <- rexp(1, 5); 0.0066770975835299 < x && x < 0.619842406933823}))

[1] 0.928